### DISCUSSION TITLE
max_grid_size with mesh refinment

### DISCUSSION THREAD

**Discussion by pordyna** [BLAST-WarpX/warpx#5480] [Q&A]
Hi!,
I see in the Amrex documentation that it is possible to set `max_grid_size` (and `max_grid_size_x` ...) for each refinement level. But not according to WarpX docs. Is it still possible? And if so, is there a way to do it with picmi?

**Comment by roelof-groenewald** [comment]
Hi @pordyna. You can set the `max_grid_size` through picmi by passing the argument `warpx_max_grid_size=...` to the grid initialization (ex. `Cartesian3DGrid(..., warpx_max_grid_size=128)`). You can also pass `warpx_max_grid_size_x` and `warpx_max_grid_size_y`. 
I've only used it with a single level so I'm not sure if it will work to pass a list of values for each refinement level. If you try it and it doesn't work, please let us know so we can extend the functionality.
